interactive_engine/compiler/src/test/java/com/alibaba/graphscope/gremlin/PathExpandStepTest.java

```diff
@@ -60,4 +58,20 @@ public void g_V_path_expand_label_test() {
         FfiNameOrId.ByValue label = op.getParams().get().getTables().get(0);
         Assert.assertEquals("knows", label.name);
     }
+
+    // g.V().out("1..2").with("PathOpt", "Simple").with("ResultOpt", "AllV")
+    @Test
+    public void g_V_path_expand_with_test() {
+        Traversal traversal =
+                g.V().out(__.range(1, 2)).with("PathOpt", "Simple").with("ResultOpt", "AllV");
```

Comment:
Can we use enum instead of string to configure these "PathOpt", "Simple", "ResultOpt", "AllV"?